cra-rxjs-styled-components/package.json

```diff
@@ -87,7 +88,6 @@
 		"@types/styled-components": "^5.1.21",
 		"eslint-plugin-react-hooks": "^4.3.0",
 		"prettier": "^2.5.1",
-		"webpack": "^5.70.0",
```

Comment:
project refused to build because this version of `webpack` was incompatible with the current version of `react-script`, so the cli recommended to remove this line and `npm install`, then the build was successful